src/components/button/index.tsx

```diff
@@ -250,12 +254,12 @@ class Button extends PureComponent<Props, ButtonState> {
 
   getBackgroundColor() {
     const {backgroundColor: themeBackgroundColor, modifiers} = this.props;
-    const {disabled, outline, link, backgroundColor: propsBackgroundColor} = this.props;
+    const {disabled, outline, link, disabledBackgroundColor, backgroundColor: propsBackgroundColor} = this.props;
     const {backgroundColor: stateBackgroundColor} = modifiers;
 
     if (!outline && !link) {
       if (disabled) {
-        return ThemeManager.CTADisabledColor;
+        return disabledBackgroundColor ? disabledBackgroundColor : ThemeManager.CTADisabledColor;
```

Comment:
`disabledBackgroundColor || ThemeManager.CTADisabledColor;` is cleaner.

Reply:
or set default `disabledBackgroundColor = ThemeManager.CTADisabledColor` in line #257.

Reply:
Right, I will change it 👍🏽